src/tile/tile.spec.ts

```diff
@@ -1,48 +1,210 @@
 import expect from "expect";
+
 import { Tile, checkIfCoordsExist, findSiblings, renderTile } from "./tile";
 import { BLACK_PLAYER } from "../player/player";
 import { getMockFromJson } from "../mocks/getMock";
 
-const board = getMockFromJson("boards/greenToGreenPath.json");
-
 describe("renderTile", () => {
   it("should render a tile", () => {
     const tile: Tile = { color: "blue", symbol: "A" };
     expect(renderTile(tile)).toMatch(/A/);
   });
 });
 
-describe("Find siblings around a tile", () => {
-  it("should find 6 sibling matching", () => {
-    const siblings = findSiblings(board, { x: 1, y: 4 });
-    expect(siblings.next).toBeDefined();
-    expect(siblings.previous).toBeDefined();
-    expect(siblings.bottomLeft).toBeDefined();
-    expect(siblings.bottomRight).toBeDefined();
-    expect(siblings.topLeft).toBeDefined();
-    expect(siblings.topRight).toBeDefined();
-  });
-  it("should find 1 sibling matching a player", () => {
+describe("Find siblings played by black around a tile", () => {
```

Comment:
Shouldn't we name that according to tested function's name ?
```suggestion
describe("findSiblings", () => {
```